source/about/self-hosted-subscriptions.rst

```diff
@@ -62,7 +73,13 @@ To add more users to your existing subscription, `contact us <https://mattermost
 
 When you subscribe to an annual plan with Mattermost, you agree to provide quarterly reports of the actual number of registered users within your system if you're above the amount you subscribed in your purchase order. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**.
 
-We'll send you an email notice around the end of the quarter reminding you to send us your report. To send this report, take a screenshot of **System Console > Site Statistics** and send in a reply email back to us. Please ensure your screenshot is taken from the top of the page and includes the **Total Active Users** metric.
+We'll send you an email notice around the end of the quarter reminding you to send us your report. 
+
+From Mattermost v7.7, you can send us this information from **System Console > License and Edition > True-up Review**. 
```

Comment:
Sorry for spotting this after the fact @justinegeffen , but a couple things:

- This feature is going to be ready for 7.9, not 7.7
- They can send the info via `System Console > Site Statistics` once prompted by their CSM

Reply:
Ok, so what's the process for 7.7? Still the same?

Reply:
I've removed "From Mattermost v7.7, you can send us this information from **System Console > License and Edition > Site Statistics**." - what is the process that's followed in 7.7? 

- Is there a bundle that can be generated?
- Or do users still need to follow the screenshot process for now?